src/PositionManager.sol

```diff
@@ -155,12 +159,12 @@ contract PositionManager is IPositionManager, ERC721Permit, PoolInitializer, Mul
 
         // the sender is the payer or receiver
         if (currencyDelta < 0) {
-            currency.settle(poolManager, sender, uint256(-int256(currencyDelta)), false);
+            _settle(currency, sender, uint256(-int256(currencyDelta)));
```

Comment:
this is very weird to me that only _settle is included in the DeltaResolver library? can we just define all functions that deal with closing deltas in 1 library?

for take its quite simple but then you dont need to pass poolManager

its just _take(currency, sender, delta)

and then later on we can also add mint/burn in that lib